Pinta.Gui.Widgets/Widgets/ToolBoxWidget.cs

```diff
@@ -11,18 +12,24 @@ public ToolBoxWidget ()
 		PintaCore.Tools.ToolAdded += HandleToolAdded;
 		PintaCore.Tools.ToolRemoved += HandleToolRemoved;
 
+		// Force images to have a specific size regardless of size of the application canvas
+		// Discussion #1374 where icons seems to be getting smaller when the screen gets bigger!
+		Gtk.CssProvider css = Gtk.CssProvider.New ();
+		css.LoadFromString (".ToolBoxWidget { -gtk-icon-size: 2rem; }"); // Works as well for high resolution and low resolution (Tested 1440P 100% scaling)
+		Gdk.Display? display = Gdk.Display.GetDefault () ?? null;
+		if (display is not null) {
+			Gtk.StyleContext.AddProviderForDisplay (display, css, 1);
+		}
+
 		SetOrientation (Orientation.Vertical);
-		ColumnSpacing = 0;
-		RowSpacing = 0;
-		Homogeneous = true;
-		MinChildrenPerLine = 6;
-		MaxChildrenPerLine = 1024; // If there is enough vertical space, only use one column
-		Vexpand = true;
-		Valign = Align.Start;
+		MinChildrenPerLine = 8; // Pinta 3 has 22 default tools, meaning a max of 3 columns regardless of size, smaller values don't lead to better use of visual space.
+		MaxChildrenPerLine = 1024;
 	}
 
 	public void AddItem (ToolBoxButton item)
 	{
+		// Despite .Flat already being set, if you use .AddCSSClass("ToolBoxWidget"), stuff doesn't work, so we set both at once.
+		item.Tool.ToolItem.SetCssClasses (["ToolBoxWidget", AdwaitaStyles.Flat]);
```

Comment:
Perhaps this could move into the `ToolBoxButton` constructor to replace its `AddCssClass(AdwaitaStyles.Flat)` call? (which doesn't do anything now since it's being overridden here)
Organization-wise, it feels cleaner to have the button set up its own style correctly

In the long run I think we'll also likely add an application-wide css file (related to #1370) but loading the custom css in the `ToolBoxWidget` constructor is fine as-is since this keeps it to a simple change that's safer to backport to a 3.0.x minor release